outputs/nexusiq.go

```diff
@@ -0,0 +1,181 @@
+package outputs
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aquasecurity/postee/v2/formatting"
+	"github.com/aquasecurity/postee/v2/layout"
+)
+
+var notAllowed = regexp.MustCompile(`[\.:\/]`)
+
+func sanitizedAppName(appName string) string {
+	return notAllowed.ReplaceAllString(appName, "_")
+}
+
+type NexusIqOutput struct {
+	Name           string
+	Url            string
+	User           string
+	Password       string
+	OrganizationId string
+}
+
+func (nexus *NexusIqOutput) GetName() string {
+	return nexus.Name
+}
+
+func (nexus *NexusIqOutput) Init() error {
+	log.Printf("Starting Nexus IQ output %q, for sending to %q", nexus.Name, nexus.Url)
+	return nil
+}
+func (nexus *NexusIqOutput) auth() string {
+	return base64.StdEncoding.EncodeToString([]byte(nexus.User + ":" + nexus.Password))
+}
+
+func (nexus *NexusIqOutput) execute(method string, url string, payload string, headers map[string]string) (map[string]interface{}, error) {
+	client := http.DefaultClient
```

Comment:
Not for this PR but in general: I think there's quite a few places we have started to see a similar instantiation of an HTTP client. I think we should soon refactor and create a standardized HTTP client to use within Postee in order to safeguard ourselves against the pitfalls of a default client (e.g. lack of sane timeouts) as missing here.

Reply:
I wouldn't over enginer that. But if some option needs to be shared let's do that

Reply:
> I wouldn't over enginer that. But if some option needs to be shared let's do that

timeout is one that comes to my mind

Reply:
only this is left - then we can merge this in. Rest looks good to me.

Reply:
you mentioned this is not for this PR

Reply:
Sorry I wasn't clear - I meant adding timeout here for now until we can figure out (if we need to) refactor HTTP client for all the other call sites.

So for now, this will suffice:
```suggestion
	client := http.DefaultClient
	client.Timeout = time.Second * 120
```